### ISSUE TITLE
storage: separate each sample into a separate file

### ISSUE THREAD

**Issue by crwilcox** [GoogleCloudPlatform/php-docs-samples#1008]
Each sample should be in a standalone file. This workitem is to track that work.

**Comment by bshaffer** [comment]
Currently in `storage`, each sample IS in a separate file: https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/storage/src

Do you mean each sample should be able to be executed by running that file?

**Comment by meredithslota** [comment]
@crwilcox Assigning to you to answer the question above — possible candidate for new PHP folks if direction is clear.

**Comment by crwilcox** [comment]
Thanks @meredithslota.

I think a smarter version of myself would have called out a specific sample that was problematic. Of course past me didnt do that...

It is possible that this is unwarranted for php though and was part of a bulk issue creation. If it is the case this is done for certain @bshaffer feel free to close. Alternatively this might just be a 'verify quick' task? We did have some languages with some cleanup to do, but it has been 18 months or so since I made this issue. So more than likely it is in great shape.
